testsupport/wait/member.go

```diff
@@ -2296,11 +2295,6 @@ func (a *MemberAwaitility) verifySecret(t *testing.T) []byte {
 }
 
 func (a *MemberAwaitility) verifyMutatingWebhookConfig(t *testing.T, ca []byte) {
-	if val := os.Getenv("skip-webhook-checks-on-setup"); val == "true" {
```

Comment:
I guess we can keep these checks for the future use and just remove the `t.Setenv("skip-webhook-checks-on-setup", "true")` above?

Reply:
+1